src/s3_api.cc

```diff
@@ -263,9 +263,8 @@ S3API::_init_api(S3API& s3_api)
 	Aws::InitAPI(s3_api.options);
 
 	Aws::Client::ClientConfiguration conf;
-	if (s3_api.region.empty() && s3_api.endpoint.empty()) {
-		err("S3 API must be enabled by specifying a region if no endpoint "
-				"override is given.");
+	if (s3_api.region.empty()) {
+		err("S3 API must be enabled by specifying a region.");
```

Comment:
I guess a region is always required now? Is that considered breaking?

Reply:
maybe it was a bug?

Reply:
I couldn't find any mention of this when I upgraded but I see something that reverts it in aws-cpp-sdk 1.10.55 https://github.com/aws/aws-sdk-cpp/pull/2295 . I upgraded to that and confirmed that no region works again.